src/zope/pagetemplate/pagetemplate.py

```diff
@@ -172,7 +170,7 @@ def write(self, text):
         # We accept both, since the text can either come from a file (and the
         # parser will take care of the encoding) or from a TTW template, in
         # which case we already have unicode.
-        assert isinstance(text, (six.string_types, six.binary_type))
+        assert isinstance(text, ((str,), bytes))
```

Comment:
Should that ben `isinstance(text, (str, bytes))`

Reply:
Already fixed it locally but forgot to commit it.